truffle-config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
       settings: {
         optimizer: {
           enabled: true,
-          runs: 200,
+          runs: 2,
```

Comment:
why the change?

Reply:
ah, that was to optimize for deployment size (good for spending conditions). but shouldn't affect the contracts in general.